cluster/addons/volumesnapshot-crds/OWNERS

```diff
@@ -0,0 +1,7 @@
+approvers:
```

Comment:
These changes should not show up as new changes in this PR.  Please resolve the rebase issues.